src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Program.cs

```diff
@@ -57,7 +54,7 @@
 if (app.Environment.IsDevelopment())
 {
     app.UseSwagger();
-    app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Company.WebApplication1 v1"));
```

Comment:
please leave in the defaults, as it provides customers guidance on customizing this. Without the parms, there's less guidance when they get started. I know it's more code, but it provides them guidance.

Reply:
I think there's confusion on the term "defaults" here. When I say "defaults" I mean, call the overload with no arguments and let the framework (Swashbuckle) default all configuration. I'm shooting for the simplest thing that works. Folks should read Swashbuckle documentation to learn how to configure it as it isn't trivial.

Reply:
How do folks feel about a comment that points to the Swashbuckle docs instead?

Reply:
I'm okay with no params and a link. I'd love no params, but folks need to be sent to the docs to know how to customize it, for sure.

Reply:
More on this - the "v1" thing - that has been confusing for a lot of folks as it "looks like" a version, but is used in the original/MVC template as the name of the OpenAPI document. So, this has been a point of confusion, and I've heard from customers during interviews that they get tripped up on it even though they've done it 100 times before. I think no-params-in-the-template could have other consequences, but I'm in agreement with @DamianEdwards that, if we could get it down to no-params, working, and clean, that's the best case.

Reply:
Defaults from Swashbuckle appear to come from https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/ConfigureSwaggerGeneratorOptions.cs#L51

Reply:
And for SwaggerUI it's here https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIBuilderExtensions.cs#L35

Reply:
Here's some more detail on the URL specification being absolute vs. relative, seemingly indicating that we really should not be manually setting it in the template to an absolute path as it causes issues https://github.com/domaindrivendev/Swashbuckle.AspNetCore/commit/a2893063ab98a088810c96457a1d61a4a35829e3#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R293